packages/core/src/jobs/job-definitions/batchEvaluations/runEvaluationJob.test.ts

```diff
@@ -179,6 +179,39 @@ describe('runEvaluationJob', () => {
       })
     })
 
+    it('increment error counter when response has an error', async () => {
```

Comment:
We were missing updating the counter of errors on an evaluation when the error happens in the response. We were checking only the result of the run. 

Errors can happen:
1. Before `runChain` occours. Errors on `runEvaluation`
2. Errors on `runChain` which are inside the `run.response`